utils/build-common.sh

```diff
@@ -114,8 +132,8 @@ build_apisix_runtime_deb() {
     fi
 
     DEBIAN_FRONTEND=noninteractive apt-get update
-    DEBIAN_FRONTEND=noninteractive apt-get install -y openresty-openssl111-dev openresty-pcre-dev openresty-zlib-dev
-
+    DEBIAN_FRONTEND=noninteractive apt-get install -y openresty-openssl111-dev openresty-pcre-dev openresty-zlib-dev cpanminus
```

Comment:
Why should I keep the dev package of openresty when I plan to switch to openssl3?

Reply:
Okay I will remove this. Must have been leftover during merge with upstream.